frontend/app/components/api/api-v3/api-v3-cache.config.ts

```diff
@@ -57,20 +54,16 @@ function apiV3CacheConfig($provide) {
       }
     };
 
-    Object.keys($http).forEach(function(key) {
-      // Decorate all fns with our cached wrapper
-      if (typeof $http[key] === 'function') {
-        wrapper[key] = function() {
-          return $http[key].apply($http, arguments);
-        };
-      } else {
-        wrapper[key] = $http[key];
-      }
+    // Decorate all fns with our cached wrapper
+    Object.keys($http).forEach(key => {
+      let prop = $http[key];
+      let fn = function() { return prop.apply($http, arguments) };
```

Comment:
Actually I am not sure wether the distinction between 'normal' properties and functions is necessary in this case.
Have you tried to just assign the value without the distinction, @oliverguenther ?